admin/cluster_containers_setup.py

```diff
@@ -0,0 +1,324 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+import yaml
+import os
+import sys
+
+from copy import deepcopy
+from json import dumps
+from itertools import repeat
+from treq import json_content
+
+from twisted.internet.defer import inlineCallbacks
+from twisted.python.filepath import FilePath
+from twisted.python import usage
+from twisted.web.http import OK
+
+from flocker.common import loop_until
+from flocker.control.httpapi import REST_API_PORT
+from flocker.apiclient import FlockerClient
+from flocker.ca import treq_with_authentication
+
+
+class ContainerOptions(usage.Options):
+    """
+    Parses the options pased as an argument to the create container script.
+    """
+    # XXX mandatory options
+    # XXX validations
```

Comment:
They are there because there is no much validation anywhere, and after speaking with Jono yesterday, it looks like there is no standard way of doing it, or standard on what to validate and what not :/ 
I added the validation that seems reasonable to me. It may be too much, or too less, or there might be better ways of doing it, but I didn't like the no validation approach.